src/nvim/version.c

```diff
@@ -57,7 +57,6 @@ static char *(features[]) = {
   "+cursorbind",
   "+cursorshape",
   "+dialog_con",
-  "+diff",
```

Comment:
Also worth mentioning is that I might have removed too many features from that branch, but my understanding is that platform parity is a big goal (hence the removal of the `+mouse_*` and X11 related items.